plugins/inputs/riemann_listener/riemann_listener.go

```diff
@@ -204,7 +204,7 @@ func (rsl *riemannListener) read(conn net.Conn) {
 			riemannReturnErrorResponse(conn, "Failed to unmarshal")
 			return
 		}
-		riemannEvents := riemanngo.ProtocolBuffersToEvents(messagePb.Events)
+		riemannEvents := protocolBuffersToEvents(messagePb.Events)
```

Comment:
Is there any reason why you need a copy of the received events? Can't you directly iterate on the events?

Reply:
We could directly iterate over the events, but since we are now using an updated fork of the `riemann` repo (i.e., not including our own copy of the `ProtocolBuffersToEvents()` function in Telegraf), I think it is cleaner to call this function in `riemanngo` instead of handling the logic from there ourselves